pmoves/tools/tz-cli/tz_cli/cli.py

```
@@ -0,0 +1,741 @@
+            sys.exit(1)
+    else:
+        backup_path = backups[0]
+        info_msg(f"Using most recent backup: {backup_path.name}")
+
+    try:
+        import shutil
+        shutil.copy2(backup_path, DEFAULT_CONFIG_PATH)
+        success_msg(f"Rolled back to: {backup_path.name}")
+        info_msg("Run 'tz reload' to apply the rollback")
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)